packages/admin-portal/src/core/__styles__/index.ts

```diff
@@ -0,0 +1,20 @@
+import { css } from 'linaria'
+
+export const WHITE = '#fff'
+export const BLACK = '#3b454e'
+export const GREY = '#74818d'
+export const GREY_LIGHT = '#f5f7f9'
+export const GREEN_LIGHTER = '#acf2bd'
+export const RED_LIGHTER = '#fdb8c0'
+export const REAPIT_MID_BLUE = '#0061a8'
```

Comment:
Could you remove the colors defined here? Seem duplicated the file at `packages/admin-portal/src/core/__styles__/colors.ts`